src/index.html

```diff
@@ -3,12 +3,89 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="main-box">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <div class="list-container">
+        <nav>
+          <ul class="nav-list">
+            <li>
+              <a
+                href="#apple"
+                class="nav-link is-active"
```

Comment:
If there are only 2 attributes and it does not exceed 80 characters with the closing tag, you can put all of that in one line :)